tools/test/test262/make_excludelist.py

```diff
@@ -30,6 +31,7 @@
 PROJECT_SOURCE_DIR = dirname(dirname(dirname(dirname(abspath(__file__)))))
 SCRIPT_SOURCE_DIR = dirname(abspath(__file__))
 DEFAULT_ESCARGOT = join(PROJECT_SOURCE_DIR, 'escargot')
+DEFAULT_EXCLUDE_LIST = "./tools/test/test262/excludelist.orig.xml"
```

Comment:
How about defining this default file path as absolute path using `SCRIPT_SOURCE_DIR` defined above?